patterns/2-structured/governance-levels.md

```diff
@@ -1,6 +1,6 @@
 ## Title
 
-Transparent Governance Levels
+Project Governance Levels
```

Comment:
Many times our patterns are related to either the InnerSource **program**, or a specific InnerSource **project**.

To make it explicit that we are talking about the governance of a specific project, we might add this to the name.
(Also we are using the term "transparent" too frequently)

Disadvantage: We may end up using to term "project" in pattern names too frequently as well, much like "InnerSource".

Reply:
The pattern also mentions the "InnerSource operating model", as an alternative phrasing to "governance levels". Maybe we should pick only of of the two and stick to it throughout the pattern?

Reply:
"Who can contribute" would be another possible name.

Reply:
Flutter calls it the "Inner Source Pyramid"

Reply:
https://github.com/InnerSourceCommons/InnerSourcePatterns/blob/11e178573c4e68e4c8036888dbafb5960a1c1fa3/meta/pattern-template.md?plain=1#L85 ... this template lists an extra section called Alias where different pattern names can be listed.

Reply:
I was thinking something like 'explicitly defined governance' or 'defined project governance'.

Reply:
I have added the Alias section, just in case we want to add some more there.

However finding the one title/name that we can stick to, is still a good thing to do. This name will be used a lot when referencing the pattern, will show up in the URL, etc. So I think it is worthwhile to burn some brainpower on this one :)

Also see [Naming Patterns](https://github.com/InnerSourceCommons/InnerSourcePatterns/blob/main/meta/pattern-style-guide.md#naming-patterns).

And yes, also here applies "naming things is hard" :)

Reply:
More funky names:
- What's my leverage?
- How much say to I get?

Reply:
Moved the Aliases we discussed here to the **Alias** section